scripts/priorities.py

```diff
@@ -31,13 +31,13 @@
         tmp["priority"] = -tmp.distance - tmp.N*args.Nweight
         name_prior = [(name, d.priority) for name, d in tmp.iterrows()]
         shuffle(name_prior)
-        candidates[focal_seq] = sorted(name_prior, key=lambda x:x[1])
+        candidates[focal_seq] = sorted(name_prior, key=lambda x:-x[1])
```

Comment:
This is a mostly stylistic point, but we could use `sorted(name_prior, key=lambda x: x[1], reverse=True)` here instead of negating the priority value. It's a more verbose solution to get the same result, but it would clearly highlight that we want to sort in descending order. By comparison, it's easy to overlook the negative sign or its function here.

Reply:
yeah. good point. I had thought that since the thing we sort is negative, the `-` might be sensible. but it is still that the highest value should end up first...